editor/project_manager.cpp

```diff
@@ -2445,9 +2449,9 @@ ProjectManager::ProjectManager() {
 	// TRANSLATORS: This refers to the application where users manage their Godot projects.
 	if (TS->is_locale_right_to_left(TranslationServer::get_singleton()->get_tool_locale())) {
 		// For RTL languages, embed translated part of the title (using control characters) to ensure correct order.
-		DisplayServer::get_singleton()->window_set_title(VERSION_NAME + String(" - ") + String::chr(0x202B) + TTR("Project Manager") + String::chr(0x202C) + String::chr(0x200E) + " - " + String::chr(0xA9) + " 2007-2021 Juan Linietsky, Ariel Manzur & Godot Contributors");
```

Comment:
Does this need some closing control char, or is it good as is?

Reply:
Since there is no LTR parts on the both sides of `TTR("Project Manager")`, control chars and the whole `if` are not necessary anymore. It can be replaced by a single line:
```c++
DisplayServer::get_singleton()->window_set_title(VERSION_NAME + String(" - ") + TTR("Project Manager"));
```